base_tier_validation/models/tier_validation.py

```diff
@@ -232,18 +247,6 @@ def _compute_hide_reviews(self):
         for rec in self:
             rec.hide_reviews = rec[self._state_field] not in self._state_from
 
-    @api.model
-    def _calc_reviews_validated(self, reviews):
```

Comment:
:warning: these methods should have been deprecated.
It breaks https://github.com/OCA/stock-logistics-workflow/tree/18.0/stock_picking_tier_validation

https://github.com/OCA/stock-logistics-workflow/actions/runs/16695540235/job/47889988766#step:8:1030

@victoralmau can you please check that?